src/sphinxcontrib/bibtex/__init__.py

```diff
@@ -50,6 +55,19 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     app.add_directive("footbibliography", FootBibliographyDirective)
     app.add_role("footcite", FootCiteRole())
 
+    # Catch bug in newer docutils
+    if (0, 18, 0, 'final', 0, True) <= docutils.__version_info__ < (0, 20, 0, 'final', 0, True):
+        error = RuntimeError(
+            (
+                "sphinxcontrib-bibtex is known to be broken with the installed version of "
+                "docutils ({}). Once a fixed version is available, please update both "
+                "packages (sphinxcontrib-bibtex and docutils) to remove this error."
```

Comment:
I'd rephrase this warning to be more clear and specific, e.g.

```Beware that docutils versions 0.18 and 0.19 (you are running {}) are known to generate invalid html for citations. If this issue affects you, please use docutils<=0.17 (or >=0.20 once released) instead. For more details, see https://sourceforge.net/p/docutils/patches/195/```

Also I think this message should only be shown when generating html; no point in bugging users who don't use the html builder.